documentation/asciidoc/computers/camera/rpicam_options_vid.adoc

```diff
@@ -1,138 +1,115 @@
 
-Pressing Enter will toggle `rpicam-vid` between recording the video stream and not recording it (i.e. discarding it). The application starts off in the recording state, unless the `--initial` option specifies otherwise. Typing `x` and Enter causes `rpicam-vid` to quit.
+* `h264` - use H.264 encoder (the default)
+* `mjpeg` - use MJPEG encoder
+* `yuv420` - output uncompressed YUV420 frames.
+* `libav` - use the libav backend to encode audio and video (for more information, see xref:camera_software.adoc#libav-integration-with-rpicam-vid[`libav`])
 
-Example: `rpicam-vid -t 0 -o test.h264 -k`
+==== `keypress`
 
-----
-	--signal,	-s		Toggle between recording and pausing when SIGUSR1 received
-----
+Alias: `-k`
 
-The SIGUSR1 signal will toggle `rpicam-vid` between recording the video stream and not recording it (i.e. discarding it). The application starts off in the recording state, unless the `--initial` option specifies otherwise. SIGUSR2 causes `rpicam-vid` to quit.
+Allows the CLI to enable and disable video output using the *Enter* key. Always starts in the recording state unless specified otherwise with xref:camera_software.adoc#initial[`initial`]. Type the `x` key and press *Enter* to exit. Does not accept a value.
 
-Example:
+==== `signal`
 
-`rpicam-vid -t 0 -o test.h264 -s`
+Alias: `-s`
 
-then 
+Allows the CLI to enable and disable video output using `SIGUSR1`. Use `SIGUSR2` to exit. Always starts in the recording state unless specified otherwise with xref:camera_software.adoc#initial[`initial`]. Does not accept a value.
 
-`kill -SIGUSR1 $!`
+==== `initial`
 
-----
-	--initial			Start the application in the recording or paused state <string>
-----
+Default value: `record`
 
-The value passed may be `record` or `pause` to start the application in, respectively, the recording or the paused state. This option should be used in conjunction with either `--keypress` or `--signal` to toggle between the two states.
+Specifies whether to start the application with video output enabled or disabled. Accepts the following values:
 
-Example: `rpicam-vid -t 0 -o test.h264 -k --initial pause`
+* `record`: Starts with video output enabled.
+* `pause`: Starts with video output disabled.
 
-----
-	--split				Split multiple recordings into separate files
-----
+Use this option with either xref:camera_software.adoc#keypress[`keypress`] or xref:camera_software.adoc#signal[`signal`] to toggle between the two states.
 
-This option should be used in conjunction with `--keypress` or `--signal` and causes each recording session (in between the pauses) to be written to a separate file.
+==== `split`
 
-Example: `rpicam-vid -t 0 --keypress --split --initial pause -o test%04d.h264`
+When toggling recording with xref:camera_software.adoc#keypress[`keypress`] or xref:camera_software.adoc#signal[`signal`], writes the video output from separate recording sessions into separate files. Does not accept a value.
```

Comment:
Should probably mention that this needs to be used in conjunction with the `output` option (e.g. the `-o test%04d.h264` in the original example), as I'm guessing otherwise it'd just keep overwriting the same file? :thinking: